package/MDAnalysis/analysis/pca.py

```diff
@@ -415,7 +417,15 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
 
         dot = np.zeros((n_frames, dim))
 
-        for i, ts in enumerate(traj[start:stop:step]):
+        if verbose == True:
+            self.disabled = False
+        else:
+            self.disabled = True
+
+        # for i, ts in tqdm(enumerate(traj[start:stop:step]), desc="Transform progress"):
+        for i, ts in tqdm(enumerate(traj[start:stop:step]), disable=self.disabled, desc="Transform progress"):
```

Comment:
Just write
```python
disable=not verbose
```

Reply:
Check PEP8: this line almost certainly needs to be broken

Reply:
Apologies. Done now.